src/librustdoc/core.rs

```diff
@@ -271,7 +272,7 @@ pub(crate) fn create_config(
         register_lints: Some(Box::new(crate::lint::register_lints)),
         override_queries: Some(|_sess, providers, _external_providers| {
             // Most lints will require typechecking, so just don't run them.
-            providers.lint_mod = |_, _| {};
+            providers.lint_mod = |tcx, module_def_id| late_lint_mod(tcx, module_def_id, MissingDoc);
```

Comment:
About this change: will it run more lints that it currently does? Or is it required to run rustdoc lints?

Reply:
This is needed to run `MissingDocs`, as that's the only lint in rustdoc that uses the late linting infra.

More generally, this will run:
- MissingDocs;
- builtin dynamic late_mod passes, there are none;
- late_mod passes registered in the lint store by rustdoc, I don't think there is any.

Reply:
Perfect, thanks for the explanations! Can you add a comment with this information on top of this change please?